### ISSUE TITLE
consideration of context attributes in <MediaConch>

### ISSUE THREAD

**Issue by dericed** [MediaArea/MediaConch_SourceCode#421]
Working on the verapdf transform and see some attributes there worth borrowing. Their implementation report starts with:
```
<report xmlns="http://www.verapdf.org/MachineReadableReport" creationDate="2016-10-15T17:36:14.985+02:00" processingTime="00:00:00.068" version="0.24.2" buildDate="2016-10-11T12:21:00+02:00">
```

Should MediaConch also add in?

- creationDate
- processingTime
- buildDate

**Comment by JeromeMartinez** [comment]
Yes

**Comment by dericed** [comment]
I don't think this could be generated within the xsl, so would have to pass them in via `--stringparam` such as `--stringparam creationDate xxx --stringparam processingTime yyy --stringparam buildDate zzz`. If that sounds ok I could xsl the xsl to use them if provided.

**Comment by JeromeMartinez** [comment]
we can create string internally and pass it to the XSL.
